base/sysimg.jl

```diff
@@ -360,10 +360,10 @@ using .I18n
 
 # frontend
 include("initdefs.jl")
-include("Terminals.jl")
-include("LineEdit.jl")
-include("REPLCompletions.jl")
-include("REPL.jl")
+include("repl/Terminals.jl")
```

Comment:
using joinpath if possible may avoid mixed forward and backslashes in backtraces on windows